### PR TITLE
Adding Library csscritic 0.6.0

### PR DESCRIPTION
**Pull request by cburgmer**
Repo: https://github.com/cburgmer/csscritic
URL: http://cburgmer.github.io/csscritic

### CODE REVIEW

File: files/csscritic/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "npm",
+  "name": "csscritic",
+  "repo": "cburgmer/csscritic",
+  "files": {
+    "basePath": "dist/",
```
Comment (jimaek) [inline files/csscritic/update.json]: I dont see a dist/ folder in the repo https://github.com/cburgmer/csscritic/tree/0.6.0

File: files/csscritic/update.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    "include": ["csscritic.allinone.js"]
```
Comment (jimaek) [inline files/csscritic/update.json]: Same for the file, its not in the repo so the bot will fail to auto-update

Comment (cburgmer) [inline files/csscritic/update.json]: Yes, but it is available from the npm package. Is this not enough? The build process generates the file and provides it in the package.

File: files/csscritic/update.json (hunk 1 above)
Comment (cburgmer) [inline files/csscritic/update.json]: See below

**Pull request merged**